k8s/helmfile/env/staging/elasticsearch.values.yaml.gotmpl

```diff
@@ -2,8 +2,6 @@ imageTag: "6.8.23-wmde.6"
 
 esJavaOpts: "-Xms2g -Xmx2g"
 
-antiAffinity: hard
```

Comment:
This deletion is a drive by, removing the production default of `hard` repeated in the staging values.